src/main/java/com/ldtteam/structures/client/BlueprintRenderer.java

```diff
@@ -77,9 +77,17 @@ private BlueprintRenderer(final BlueprintBlockAccess blockAccess)
         init();
     }
 
+    public void updateBlueprint(final Blueprint blueprint)
```

Comment:
jdoc

Reply:
Just plain nope -> One renderer per Blueprint, is architecturally way better. Recreate it if you need to and evict the renderer from your cache!